test-apps/appui-test-app/appui-test-providers/src/ui/frontstages/ComponentExamples.tsx

```diff
@@ -25,8 +27,8 @@ export interface ComponentExampleCategory {
  */
 export class ComponentExamplesModalFrontstage implements ModalFrontstageInfo {
   public static stageId = "appui-test-app:componentExamplesStage";
-  public title: string = UiFramework.localization.getLocalizedString(
-    "SampleApp:componentExamplesStage.examples"
+  public title = AppUiTestProviders.translate(
```

Comment:
This is analogous to `useRef` in a functional component.
I.e. this will be called once to initialize the instance. Move this to `render`. Not adding comments for other similar cases.

Reply:
Good suggestions just not in this case as this member is inherited from `ModalFrontstageInfo` and cannot be moved. Applied this change for classes where possible though.

Reply:
whoops, sorry about that, mistaken this for a class component :)